drivers/interrupt_controller/intc_plic.c

```diff
@@ -22,12 +22,15 @@
 #include <zephyr/irq.h>
 
 #define PLIC_BASE_ADDR(n) DT_INST_REG_ADDR(n)
+#define PLIC_REG_PRIO_OFFSET 0x0
```

Comment:
I believe we should add a comment here saying that these offsets are taken from the RISC-V PLIC Specification, so that it will be clear in the future that these offsets should be defined here in the driver, and not in the device tree as they are not configurable.